hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/utils/SynthTxnUtils.java

```diff
@@ -155,7 +155,6 @@ public static boolean hasNonDegenerateAutoRenewAccountId(@NonNull final Account
      */
     public static CryptoCreateTransactionBody synthHollowAccountCreation(@NonNull final Bytes evmAddress) {
         requireNonNull(evmAddress);
-        // TODO - for mono-service equivalence, need to set the initial balance here
         return CryptoCreateTransactionBody.newBuilder()
```

Comment:
(So apparently this comment was _not_ removed when the code was changed to set the initial balance?)

Reply:
apparently